### DISCUSSION TITLE
Multi-select + delete, Black and White mode

### DISCUSSION THREAD

**Discussion by gsturov** [elfmz/far2l#2279] [Q&A]
First of all, far2l is insanely cool and amazingly great.  :)

My version: version 2.6.1-2024-06-26-64af534-beta Linux x86_64

#1 I multi-select a bunch of folders and try to delete them. Sometimes I got only one deleted (and the rest are still selected), sometimes a few folders got deleted, sometimes all of them got deleted.

#2 I select "Black and White" mode, save settings, quit far2l, start it again, and it is in color mode.

Known problems? Fixed already? Should I report them as "Issues"?

thanks!

**Comment by spnethw** [comment]
I think you should create the issue for №1 giving as much details as possible:
1. filesystem type?
2. whether the names of these folders and the files in them contained unusual characters?
3. whether these folders contained special files?
4. whether the deletion was made to Trash or directly?
5. did the far2l display any errors?
6. was privilege elevation (sudo) performed?

**Comment by gsturov** [comment]
1. ext4
2. no
3. no
4. Recycle bin is DISABLED
5. no errors
6. no sudo

It is difficult to reproduce. I ran this scenario ~30 times and 2 times I saw the problem. And then I tried to reproduce it with a screen recorder and I was not able to.
